packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { HTMLProps, MouseEvent } from 'react';
+import { jsx, css } from '@emotion/core';
+
+import Body from '../Body';
+import Headline from '../Headline';
+import ButtonGroup from '../ButtonGroup';
+import Button from '../Button';
+import { spacing } from '../../styles/style-mixins';
+import Image from '../Image';
+
+type Action = {
+  onClick?: (event: MouseEvent<HTMLButtonElement | HTMLAnchorElement>) => void;
+  href?: string;
+  text: string;
+};
+
+export interface NotificationFullscreenProps extends HTMLProps<HTMLDivElement> {
+  /**
+   * A concise description of the imageSrc prop.
+   */
+  imageSrc: string;
```

Comment:
I saw your conversation with Giao-Chung in Figma regarding the alternative text for the image. As I explained there, we should add the ability to provide alt text. I propose turning the prop into an object: 

```suggestion
  /**
   * A concise description of the image prop.
   */
  image: { src: string; alt: string };
```

(I made the `alt` prop required so developers have to think about passing text or an empty string to hide the image from screen readers).